### PR TITLE
[IMP] content/developer/webservices

### PR DESCRIPTION
**Pull request by Julien00859**
It is hard to find and learn about RPC in Odoo, users hardly know about
JSON-RPC as the only place where we talk about it in deep down in the
*How to make a Module* tutorial.

In the commit, we isolated the tutorial about RPC in a dedicated file,
that file is better referenced thanks to cross-references in some others
places that are talking about RPC

Task: 2884559

Rebased to 15.0 from master #2427

### CODE REVIEW

File: content/developer/api/external_api.rst [hunk 1]
```diff
@@ -7,6 +7,14 @@ all of its data are also available from the outside for external analysis or
 integration with various tools. Part of the :ref:`reference/orm/model` API is
 easily available over XML-RPC_ and accessible from a variety of languages.
 
+.. seealso::
+   - :doc:`Tutorial on web services <../howtos/web_services>`
+
+.. warning::
```
Comment (AntoineVDV) [inline content/developer/api/external_api.rst:13]: This is not exactly a warning; more like an important note.
```suggestion
.. important::
```

Comment (AntoineVDV) [reply content/developer/api/external_api.rst:13]: Also I'd swap it with the `seealso`.

File: content/developer/api/external_api.rst [hunk 2]
```diff
@@ -7,6 +7,14 @@ all of its data are also available from the outside for external analysis or
...
+   Starting PHP8, the XML-RPC extension may not be available by default,
+   checkout the `manual <https://www.php.net/manual/en/xmlrpc.installation.php>`_
+   for installation steps.
```
Comment (AntoineVDV) [inline content/developer/api/external_api.rst:16]:
```suggestion
   Starting with PHP8, the XML-RPC extension may not be available by default.
   Check out the `manual <https://www.php.net/manual/en/xmlrpc.installation.php>`_
   for the installation steps.
```

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

**Comment by AntoineVDV** [comment]
@robodoo r+ rebase-ff

**Comment by jcs-odoo** [comment]
It looks like this r+ didn't go through...
@AntoineVDV @Julien00859

**Comment by AntoineVDV** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**